packages/manager/modules/sign-up/src/form/details/details.html

```diff
@@ -273,4 +272,25 @@
             <!-- /input[name="city"] -->
         </div>
     </div>
+
+    <div class="row">
+        <div class="col-lg">
+            <!-- purchase purpose -->
+            <oui-field
+                data-label="{{ 'sign_up_details_field_purpose_of_purchase' | translate }}"
+            >
+                <input
+                    type="text"
+                    class="oui-input"
+                    name="purposeOfPurchase"
+                    id="purposeOfPurchase"
+                    maxlength="{{:: $ctrl.PURPOSE_OF_PURCHASE_MAX_LENGTH }}"
+                    data-ng-model="$ctrl.signUpFormCtrl.model.purposeOfPurchase"
+                    data-ng-required="$ctrl.signUpFormCtrl.rules.purposeOfPurchase.mandatory"
```

Comment:
Since purposeOfPurchase is not given by the api, i think $ctrl.signUpFormCtrl.rules.purposeOfPurchase does not exist and so it is not necessary to add this ng-required and the following ng-pattern

Reply:
Hello @JacquesLarique API changes had been made to add this field only to the subsidiary IN. You wont see it for any other subsidiary.